pkg/uhttp/wrapper_test.go

```diff
@@ -137,7 +137,7 @@ func TestWrapper_NewRequest(t *testing.T) {
 			}
 
 			client := BaseHttpClient{
```

Comment:
It'd be nice if the test used the init function here.

Reply:
Will do